src/Microsoft.AspNet.TestHost/ResponseMessage.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.TestHost
+{
+    class ResponseMessage : HttpResponseMessage
```

Comment:
All classes need the visibility. Please see our [Engineering Guidelines](https://github.com/aspnet/Home/wiki/Engineering-guidelines)